packages/as-transformer/src/helpers/protobuf.ts

```diff
@@ -126,6 +152,15 @@ function getTypeName(type: string): FieldSpec {
   return spec;
 }
 
+/**
+ * Generates the AS helpers code for the passed protobuf file.
+ *
```

Comment:
Could be cool to have an short explaination of what is an helper in this context, why we need it.